color.js

```diff
@@ -12,11 +12,23 @@ export class Color {
   }
 
   //"Factory method" that creates color from given HEX value
-  static fromHex(hexColor) {
+  static hexToRGB(hexColor) {
+    const r = parseInt(hexColor.slice(1, 3), 16);
+    const g = parseInt(hexColor.slice(3, 5), 16);
+    const b = parseInt(hexColor.slice(5, 7), 16);
+    const res = { r, g, b };
+    return res;
   }
 
   //Converts color to HEX format
-  toHex() {
+  rgbToHex(rgbColor) {
+    let hex = '0x';
```

Comment:
Here we should use # for the start of color because of the format required by CSS (like "#ffffff")